src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,73 @@
+import * as Dialog from '@radix-ui/react-dialog';
+import { createStyles } from '../../theme/utils';
+import { keyframes } from '@emotion/react';
+import Button from '../Button/Button';
+import { Typography } from '../Typography/Typography';
+import XIcon from '../../assets/icons/XIcon';
+type ModalProps = {
+  onClick?: () => void;
+  isOpen?: boolean;
+  handleOpen?: () => void;
+};
+
+const Modal = ({ onClick, isOpen, handleOpen }: ModalProps) => (
+  <Dialog.Root open={isOpen} onOpenChange={handleOpen}>
+    <Dialog.Portal>
+      <Dialog.Overlay css={DialogStyles.overlay} />
+      <Dialog.Content css={DialogStyles.content}>
+        <Dialog.Title css={{ paddingTop: 15 }}>
+          <Typography weight="bold" size="3xl">
+            Are you sure to delete this item?
```

Comment:
it would be better to rename this component into ConfirmationModal and make the typography text be taken from props